hathor/manager.py

```diff
@@ -703,6 +710,35 @@ def generate_parent_txs(self, timestamp: Optional[float]) -> 'ParentTxs':
         can_include = [i.data for i in can_include_intervals]
         return ParentTxs(max_timestamp, can_include, must_include)
 
+    def get_best_blockchain(self, n_blocks: int) -> list['BlockInfo']:
+        """ Get a list with N blocks from the best blockchain
+        in descending order.
+        """
+        if not (0 < n_blocks <= settings.MAX_BEST_BLOCKCHAIN_BLOCKS):
```

Comment:
This is an internal method. We can add this protection on the p2p protocol and let this internal API receive any value for `n_blocks`. Maybe just add `assert n_blocks > 0`.

Reply:
Ok. Resolved.